tests/python/test_ndarray.py

```diff
@@ -231,6 +231,9 @@ def func(a: ti.any_arr(element_shape=(10, ))):
     func(v)
     assert ti.get_runtime().get_num_compiled_functions() == 1
     import torch
-    v = torch.zeros((7, 10), dtype=torch.int32)
+    v = torch.zeros((6, 11), dtype=torch.int32)
```

Comment:
Please feel free to ignore if we already have one ;) : it might also worth adding a test that triggers element_dim/layout `ValueError`s above in `ArgAnyArray` so that we make sure it's properly handled.

Reply:
Helpful suggestion! Done.